mage_ai/server/app.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask, render_template, request
 from flask_cors import CORS
 from mage_ai.data_cleaner.data_cleaner import analyze, clean as clean_data
-from mage_ai.data_cleaner.pipelines.base import BasePipeline
+from mage_ai.data_cleaner.pipelines.base import DEFAULT_RULES, BasePipeline
```

Comment:
same here. remove unused imports.

Reply:
@skunichetty i left one last comment here.

Reply:
Oops, didn't see. I'll fix that now.